jib-core/src/main/java/com/google/cloud/tools/jib/image/Image.java

```diff
@@ -44,11 +47,16 @@
     @Nullable private Instant created;
     @Nullable private ImmutableList<String> entrypoint;
     @Nullable private ImmutableList<String> programArguments;
+    @Nullable private DockerHealthCheck healthCheck;
```

Comment:
Should healthcheck be nullable still considering there's now two ways to say inherit healthcheck (set to `null` or build with `DockerHealthCheck.builderWithInheritedCommand().build()`)?

Reply:
Yeah I'm trying to decide between making it nullable in place of either `disabled()` or `builderWithInheritedCommand().build()` vs. making it not nullable and instantiating with a default value. I think you're right though and using a default will be simpler.